dali/operators/decoder/nvjpeg/nvjpeg_decoder_decoupled_api_test.cc

```diff
@@ -225,7 +225,10 @@ class HwDecoderUtilizationTest : public ::testing::Test {
     auto node = pipeline_.GetOperatorNode(decoder_name_);
     if (!node->op->GetDiagnostic<bool>("using_hw_decoder")) {
       PrintDeviceInfo();
-      if (nvml::isHWDecoderSupported()) {
+      float driver_version = nvml::GetDriverVersion();
+      // HW decoder is disabled for drivers < 455.x, see
+      // dali/operators/decoder/nvjpeg/nvjpeg_decoder_decoupled_api.h for details
+      if (driver_version >= 455 && nvml::isHWDecoderSupported()) {
```

Comment:
Perhaps we should have a function `ShouldUseHWDecoder()` which aggregates these two checks? I see this code repeated 4 times.

Reply:
Good point, will do.